bigquery_etl/cli/query.py

```diff
@@ -240,7 +241,7 @@ def schedule(name, sql_dir, project_id, dag, depends_on_past, task_name):
 
     sql_dir = Path(sql_dir)
 
-    dags = get_dags(None, sql_dir.parent / "dags.yaml")
+    dags = DagCollection.from_file(sql_dir.parent / "dags.yaml")
```

Comment:
`get_dags` parses all DAG and task definitions. Since we do not use any of the parsed task definitions here, we can use `DagCollection.from_file` which parses only the DAG definitions.